docs/contributing/standards.md

```diff
@@ -0,0 +1,69 @@
+---
+id: standards
+title: Contributing
+sidebar_label: Standards
+---
+
+# Standards
+
+The following standards are used by FAST-DNA contributors for consistency across this repository.
+
+## Core principles
+
+* Build for maintainability
+* Build for scalability
+* Build for performance
+
+## Naming Conventions
+
+* Avoid all abbreviations, except where common industry acronyms
+* Use names that are clear and intuitive
+* Use semantic names instead of presentational
+
+## Code commenting
+
+* For work to complete later use `// TODO <issue number>: <title>` and file new issue
+* For work considered a hack use `// HACK <issue number>: <title>` and file new issue
+* Be verbose and do explain what you're doing and why you're doing it
+* Use `//` for single line comment blocks
+* Use `/* */` for multiple line comment blocks
+
+## Accessibility
+
+FAST-DNA has worked diligently from the ground up to enable accessibility in collaboration with legal, accessibility, and marketing teams at Microsoft. As an ecosystem to accelerate the building of Web sites and apps we ensure you have the tools to build accessible user exerpiences.
+
+To learn more about Microsoft's commitment visit their [accessibility](https://www.microsoft.com/en-us/accessibility) website.
+
+## Browsers
+
+Component packages support IE11+ and all modern browsers. Click on the links below to report browser bugs.
+
+* [Edge](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/)
+* [Firefox](https://bugzilla.mozilla.org/)
+* [Safari](https://bugs.webkit.org/)
+* [Chrome](https://bugs.chromium.org/p/chromium/issues/list)
+* [Opera](https://bugs.opera.com/wizard/desktop)
```

Comment:
Per the other pull request comment why do we have bug reporting for browsers

Reply:
Because it's useful.  If you're testing and discover a bug, then determine it's a browser bug, there is no harm in providing places to file those bugs.  It's not easy to find all of these on the web.  In this case I prefer to have these in here.